.github/workflows/test.yml

```diff
@@ -151,7 +155,7 @@ jobs:
         run: |
           source install/setup.bash && \
           colcon test \
-            --packages-select ddsrouter \
+            --packages-select-regex ddsrouter \
```

Comment:
We can discuss this but I'd see better to separate the tests of each package in a different steps. That way we can se at a glance which package is failing. (This also applies for windows testing)